vanilla-framework/vanilla-dashboard-theme
# Table sorting

## Done

Added table sorting pattern as per https://github.com/vanilla-framework/vanilla-dashboard-theme/issues/21
Spec and design: https://github.com/ubuntudesign/vanilla-dashboard-theme-design/tree/master/Table%20sorting

## QA

- Pull the branch
- Run `./run serve`
- Open http://0.0.0.0:8102/vanilla-dashboard-theme/examples/patterns/table-sortable/table-sortable/
- Check against design: https://github.com/ubuntudesign/vanilla-dashboard-theme-design/blob/master/Table%20sorting/Table%20sorting.png
